src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -89,13 +90,13 @@
 
     @Option(name = "-keepStopwords", usage = "boolean switch to keep stopwords")
     public boolean keepStopwords = false;
-  
+
     @Option(name = "-stemmer", usage = "Stemmer: one of the following porter,krovetz,none. Default porter")
     public String stemmer = "porter";
 
     @Option(name = "-uniqueDocid", usage = "remove duplicated documents with the same doc id when indexing. " +
-      "please note that this option may slow the indexing a lot and if you are sure there is no " +
-      "duplicated document ids in the corpus you shouldn't use this option.")
+        "please note that this option may slow the indexing a lot and if you are sure there is no " +
+        "duplicated document ids in the corpus you shouldn't use this option.")
     public boolean uniqueDocid = false;
```

Comment:
this diff shouldn't be here?

Reply:
Oh, hrm, is this auto code formatting?

Reply:
Yeah, auto code formatting - I can remove it, but it's standard Intellij indenting.

Reply:
nah that's fine, let's go with IntelliJ